src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int FIGURE_PROPERTY = 10;
+    public static final int FIGURE_DEFAULT = 10;
+    public static final String COLOR_DEFAULT = "WHITE";
+    private Random random = new Random();
+    private Figure figure;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT + 1);
+
+        switch (figureNumber) {
+
+            case 1: {
+                int radius = random.nextInt(FIGURE_PROPERTY);
+                String color = colorSupplier.getRandomColor();
+                figure = new Circle(radius, color);
+                break;
+            }

```

Comment:
```suggestion
            case 0: 
                return getRandomCircle();
```
in all programming languages, you should start from 0

lets add private methods to create each figure - this would make your code easier to read